config/versions.json

```
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/ACCESS-NRI/schema/blob/main/au.org.access-nri/model/deployment/config/versions/2-0-0.json",
     "spack": "0.22",
-    "spack-packages": "2024.08.09",
-    "spack-config": "2024.07.05"
+    "spack-packages": "2025.07.006",
```

```suggestion
    "spack-packages": "2025.08.000",
```

Needs to be updated to https://github.com/ACCESS-NRI/spack-packages/releases/tag/2025.08.000 to get the change to the `gcom` package which adds `openmpi` as a direct dependency:

https://github.com/ACCESS-NRI/spack-packages/commit/ca73ffdfe7c5b0e9170048b0a504e1f0207cd68a

---

I see. I had mistakenly updated `spack-versions` instead of `spack-packages`.